event-manager/src/main/java/evrentan/community/eventmanager/controller/EventController.java

```diff
@@ -0,0 +1,58 @@
+package evrentan.community.eventmanager.controller;
+
+import evrentan.community.eventmanager.dto.EventDto;
+import evrentan.community.eventmanager.service.EventService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+
+/**
+ * REST Controller for community related API end-points.
```

Comment:
correct javadoc texts. for example, this isi not related to community, it should be event.